lib/pacio_test_kit/pfe/collection_observation_group.rb

```diff
@@ -17,6 +18,16 @@ class CollectionObservationGroup < Inferno::TestGroup
       }
       run_as_group
 
+      test from: :pacio_resource_create,
+           title: 'Server creates correct Observation resource from Observation create interaction',
+           config: {
+             inputs: {
+               resource_inputs: {
+                 name: :collection_observation_resource_list,
+                 title: 'Resources(s) for PFECollection resources created on the server'
+               }
+             }
+           }
```

Comment:
Create test is not needed here. we will only have create test for one observation profile not all, as the check just verify that we can create a given resource. Only add the create test to one `Observation` profile and one of the `DiagnosticReport` since those two resources have a `SHALL` requirement for create in the capability statement.